doc/guides/west/manifest.rst

```diff
@@ -735,33 +735,30 @@ The ``import`` key can also contain a mapping with the following keys:
 
 - ``file``: The name of the manifest file or directory. This defaults
```

Comment:
I guess, `file` is also optional.
And if not provided, `west.yml` is used.

Reply:
> I guess, file is also optional.

I'll add the word Optional, thanks.

> And if not provided, west.yml is used.

That's what it says on this line and the next one, no?

"This defaults to west.yml if not present"

Reply:
> > And if not provided, west.yml is used.
> 
> That's what it says on this line and the next one, no?
> 
> "This defaults to west.yml if not present"

Yep, I only referenced that line, to show that you indirectly stated that it was optional.
As something you had already written.